pkg/bgpv1/manager/reconcile.go

```diff
@@ -151,6 +161,11 @@ func (r *PreflightReconciler) Reconcile(ctx context.Context, p ReconcileParams)
 				AdvertiseInactiveRoutes: true,
 			},
 		},
+		OnFIBEvent: func() {
+			if r.signaler != nil {
```

Comment:
Maybe this is impossible? We set `r.signaler` on initialization.

Reply:
This is nice for unit testing.

Reply:
How about injecting a dummy signaler on testing then?